### PR TITLE
Unify log tags.

### PR DESCRIPTION
**Pull request by charlesmchen-signal**
Pretty mechanical.

PTAL @michaelkirk

### CODE REVIEW

File: Signal/src/Jobs/MultiDeviceProfileKeyUpdateJob.swift [hunk 1]
```diff
@@ -14,8 +14,6 @@ import SignalMessaging
  */
 @objc public class MultiDeviceProfileKeyUpdateJob: NSObject {
 
-    let TAG = "[MultiDeviceProfileKeyUpdateJob]"
```
Comment (charlesmchen-signal) [inline Signal/src/Jobs/MultiDeviceProfileKeyUpdateJob.swift]: Change 1: remove antiquated `TAG`s.

File: Signal/src/ViewControllers/ContactViewController.swift [hunk 2]
```diff
@@ -573,7 +573,7 @@ class ContactViewController: OWSViewController, ContactShareViewHelperDelegate {
         Logger.info("\(self.logTag) \(#function)")
 
         guard let url = NSURL(string: "tel:\(phoneNumber.phoneNumber)") else {
-            owsFail("\(ContactViewController.logTag) could not open phone number.")
+            owsFail("\(ContactViewController.logTag()) could not open phone number.")
```
Comment (charlesmchen-signal) [inline Signal/src/ViewControllers/ContactViewController.swift]: 2. Ensure all usages of class `logTag` property are invoked.

File: SignalServiceKit/src/Util/NSObject+OWS.h [hunk 3]
```diff
@@ -10,7 +10,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 @property (nonatomic, readonly) NSString *logTag;
 
-+ (NSString *)logTag;
+@property (class, nonatomic, readonly) NSString *logTag;
```
Comment (charlesmchen-signal) [inline SignalServiceKit/src/Util/NSObject+OWS.h]: 3. Convert class method to class property.

File: Signal/src/ViewControllers/ContactViewController.swift (hunk 2 above)
Comment (michaelkirk-signal) [reply Signal/src/ViewControllers/ContactViewController.swift]: hmmm... isn't this obviated by:

```
3. Convert class method to class property.
```

Comment (charlesmchen-signal) [reply Signal/src/ViewControllers/ContactViewController.swift]: Unfortunately, no.  I haven't looked into why yet.

**Pull request merged**